spec/ddtrace/contrib/concurrent_ruby/integration_test_spec.rb

```diff
@@ -4,57 +4,52 @@
 require 'ddtrace'
 
 RSpec.describe 'ConcurrentRuby integration tests' do
-  around do |example|
-    unmodified_future = ::Concurrent::Future.dup
-    example.run
+  # DEV We save an unmodified copy of Concurrent::Future.
+  let!(:unmodified_future) { ::Concurrent::Future.dup }
```

Comment:
This is a smart trick we've been doing to unpatch `concurrent-ruby`, too smart for my taste I'd say :)

Reply:
Yeah I remember running into this and it messing up my other writer extraction PR... not sure if it helps it sidestep the problem it had. Either way, not thrilled with the complexity in this test.